pkg/conversion/queryparams/convert_test.go

```diff
@@ -131,6 +138,19 @@ func TestConvert(t *testing.T) {
 			},
 			expected: url.Values{"str": {""}, "namedStr": {"named str"}},
 		},
+		{
+			input: &baz{
+				Ptr:  intp(5),
+				Bptr: boolp(true),
+			},
+			expected: url.Values{"ptr": {"5"}, "bptr": {"true"}},
+		},
+		{
+			input: &baz{
+				Bptr: boolp(true),
+			},
+			expected: url.Values{"ptr": {""}, "bptr": {"true"}},
```

Comment:
hmm, not sure that "&lt;nil&gt;" is better than an empty string. Also, can you add a test case for when the field has omitempty and is nil?